lib/core/src/sync/model/data.rs

```diff
@@ -31,6 +32,9 @@ impl ChainSyncData {
         for field in updated_fields {
             match field.as_str() {
                 "accept_zero_conf" => self.accept_zero_conf = other.accept_zero_conf,
+                "accepted_receiver_amount_sat" => {
```

Comment:
Can you add into the `updated_fields` fn below a `accepted_receiver_amount_sat` comparison to make sure we sync when it changes

Reply:
Thanks! When rebasing I forgot to update this. Fixed in [b18ad6d](https://github.com/breez/breez-sdk-liquid/pull/615/commits/b18ad6d097db46630e1e48d82944afeed8046b80)